scripts/kurtosis/network_params.json

```diff
@@ -16,7 +15,7 @@
         },
         {
             "el_client_type": "REPLACE_WITH_BUILDER",
-            "el_client_image": "ethpandaops/flashbots-builder:main-45e865e",
+            "el_client_image": "flashbots/builder:1.13.2.4844.dev5-4d161de",
```

Comment:
Is this a recent buidler version or what exactly?

Reply:
IMHO latest should be used to baseline - if not then each time version is released someone would need to post additional PR and at some point they may forgot about it

Reply:
It doesn't matter right now, currently we need a static version so we can accurately compare our implementation and be sure it's working.

As for the prod solution we need to consider advantages and disadvantages: e.g. what using latest means for comparability between prs? What it means for replayability?

Changing builder version will affect our metrics like the block inclusion rate etc. 

That is beyond the scope of this PR however we'll have to discuss baseline update on production too